src/SB/Game/zMain.cpp

```diff
@@ -608,6 +642,92 @@ void zMainLoop()
     U32 preinit_bubble_matfx[6];
     U32 preinit_shiny_models[8];
     U32 preinit_ADC_models[10];
+
+    zMainShowProgressBar();
+    xMemPushBase();
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+    xSTPreLoadScene(0x464f4e54, NULL, 0x1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
+    xSTQueueSceneAssets(0x464f4e54, 1);
+    time = iTimeGet();
+    xUtil_idtag2string(0x464f4e54, 0);
+    iTimeDiffSec(time);
```

Comment:
Please replace usages of 0x464f4e54 with the char literal representation `'FONT'`, which is more readable and more likely to be what Heavy Iron used here.